frontend/src/pages/Layout.vue

```diff
@@ -1,16 +1,29 @@
 <template>
     <VApp>
-        <Navbar />
-        <RouterView />
+        <Header :is-logged="isLogged" />
```

Comment:
with this approach you breaking single responsibility. Layout should be responsible just for layout, but not for auth guard